user-manual/access-content/browse.rst

```diff
@@ -1146,18 +1152,26 @@ redirect you to a new browse page, described below.
    more information on the fields available and their uses, see:
    :ref:`term-data-entry`.
 6. Below the :term:`treeview`, addtional :term:`facet filters <facet filter>` that
-   can be applied to the browse results are listed. For more information on using
-   facet filters in AtoM, see: :ref:`recurring-facet-filters`.
+   can be applied to the browse results are listed, if available. For more 
+   information on using facet filters in AtoM, see: :ref:`recurring-facet-filters`.
 7. If a user is authenticated (i.e. logged in) and has edit permissions, a
    :term:`button block` will be visible on the page, allowing the user to edit,
    delete, or add new terms. For more general information on working with terms
    in AtoM, see: :ref:`terms`, and especially :ref:`term-data-entry`. Public
    users who are not logged in will not see the button block.
-8. Below the term information, :term:`archival descriptions
-   <archival description>` that are linked to the current term as
-   :term:`access points <access point>` are displayed as results. Clicking on a
-   description result will redirect the user to the :term:`view page` for that
-   result.
+8. At the top of the page, result tabs will be available, showing a count
+   of related :term:`archival descriptions <archival description>` and 
+   :term:`authority records <authority record>`, where the current term has 
```

Comment:
Note that in the current qa/1.x branch and in your screencap, we are still showing these tab labels as "Related Descriptions" and "Related Authorities". However, we have an open ticket questioning whether to change these https://projects.artefactual.com/issues/13332. We can't use user interface labels as these are in singular form. However, hard-coding "Related Descriptions" and "Related Authorities" does not seem ideal either.

Reply:
Ahhh damn. Sorry, I thought the feature was stable enough to document, but it sounds like I should wait and update this PR once the labels have been addressed.